jit-compiler/src/main/java/org/aya/compiler/ModuleSerializer.java

```diff
@@ -64,7 +65,7 @@ private void doSerialize(ModuleResult unit) {
       IterableUtil.forEach(unit.defs, this::appendLine, this::doSerialize));
   }
 
-  @Override public ModuleSerializer serialize(ModuleResult unit) {
+  @Override public ModuleSerializer serialize(@NotNull FreeCodeBuilder builder, ModuleResult unit) {
```

Comment:
I noticed that the @NotNull annotation is violated in FileSerializer. Was this intentional?

Reply:
The same violation pattern happens almost everywhere. I think it's just because the code is WIP